x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/rest/action/RestMonitoringBulkAction.java

```diff
@@ -33,20 +35,28 @@
 import static org.elasticsearch.rest.RestRequest.Method.POST;
 import static org.elasticsearch.rest.RestRequest.Method.PUT;
 
-public class RestMonitoringBulkAction extends MonitoringRestHandler {
+public class RestMonitoringBulkAction extends XPackRestHandler {
 
     public static final String MONITORING_ID = "system_id";
     public static final String MONITORING_VERSION = "system_api_version";
     public static final String INTERVAL = "interval";
+    private static String URI_BASE = "/monitoring";
```

Comment:
I think the endpoint should be `/_monitoring/bulk` otherwise `/monitoring/_bulk` will conflict with existing indices named `monitoring`. We long ago reserved leading underscores from index names.

Reply:
@jasontedor that makes sense.  I have updated the PR to use `/_monitoring/bulk`